nixos/modules/services/networking/mihomo.nix

```diff
@@ -54,6 +55,16 @@ in
   };
 
   config = lib.mkIf cfg.enable {
+    assertions = [
```

Comment:
Why not drop the null default from the config file instead?

Reply:
An assertion prevents you from building the config, so the outcome is the same and it is probably a very strong hint, that you need to set that option